### ISSUE TITLE
On screen keyboard doesn't work to type in password at lock screen

### ISSUE THREAD

**Issue by lsitongia** [linux-surface/linux-surface#812]
[Description of the bug or feature]

I disconnect the type cover, rotate the screen to portrait, suspend with the power button, resume with power button.
Touch the screen and the unlock prompt comes up. Tap in the box, and the on screen keyboard comes up, at the bottom of the screen.
The keyboard doesn't work. I can slide up and down on the screen and make the keyboard disappear and reappear. Tapping in the box. Can't get the keyboard to work. Once in a while, tapping on the glyph on the lower right, or doing something (I'm not sure), the screen changes to a larger password input box, different font, and a different, more rounded, keyboard comes up. That one WORKS.

### Environment

- Hardware model: Surface Go 2
- Kernel version: Linux surface 5.17.5-surface #2 SMP PREEMPT Fri Apr 29 17:07:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
- Distribution: Ubuntu 22.04

<details><summary>`dmesg` output</summary>

(Hope it's okay that I used tail on the syslog, so that I could start it when I am about to demonstrate this problem.)

May  7 14:50:14 surface ModemManager[615]: <info>  [sleep-monitor] system is about to suspend
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.7842] manager: sleep: sleep requested (sleeping: no  enabled: yes)
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.7843] device (p2p-dev-wlp1s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.7847] manager: NetworkManager state is now ASLEEP
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.7848] device (wlp1s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
May  7 14:50:14 surface dbus-daemon[514]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.5' (uid=0 pid=515 comm="/usr/sbin/NetworkManager --no-daemon ")
May  7 14:50:14 surface systemd[1]: Starting Network Manager Script Dispatcher Service...
May  7 14:50:14 surface dbus-daemon[514]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
May  7 14:50:14 surface systemd[1]: Started Network Manager Script Dispatcher Service.
May  7 14:50:14 surface kernel: [102085.629048] wlp1s0: deauthenticating from dc:bf:e9:ff:26:12 by local choice (Reason: 3=DEAUTH_LEAVING)
May  7 14:50:14 surface gnome-shell[6272]: cr_parser_new_from_buf: assertion 'a_buf && a_len' failed
May  7 14:50:14 surface gnome-shell[6272]: cr_declaration_parse_list_from_buf: assertion 'parser' failed
May  7 14:50:14 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-DISCONNECTED bssid=dc:bf:e9:ff:26:12 reason=3 locally_generated=1
May  7 14:50:14 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-DSCP-POLICY clear_all
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9033] device (wlp1s0): supplicant interface state: completed -> disconnected
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9037] device (wlp1s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
May  7 14:50:14 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-SIGNAL-CHANGE above=0 signal=-9999 noise=9999 txrate=0
May  7 14:50:14 surface avahi-daemon[510]: Withdrawing address record for 2601:281:8300:2d40::16 on wlp1s0.
May  7 14:50:14 surface avahi-daemon[510]: Withdrawing address record for 2601:281:8300:2d40:c07c:b8d2:c970:499f on wlp1s0.
May  7 14:50:14 surface avahi-daemon[510]: Withdrawing address record for 2601:281:8300:2d40:4886:f43a:2d43:a663 on wlp1s0.
May  7 14:50:14 surface avahi-daemon[510]: Leaving mDNS multicast group on interface wlp1s0.IPv6 with address 2601:281:8300:2d40:4886:f43a:2d43:a663.
May  7 14:50:14 surface avahi-daemon[510]: Joining mDNS multicast group on interface wlp1s0.IPv6 with address fe80::32cd:3799:f3e6:f97b.
May  7 14:50:14 surface avahi-daemon[510]: Registering new address record for fe80::32cd:3799:f3e6:f97b on wlp1s0.*.
May  7 14:50:14 surface avahi-daemon[510]: Withdrawing address record for fe80::32cd:3799:f3e6:f97b on wlp1s0.
May  7 14:50:14 surface avahi-daemon[510]: Leaving mDNS multicast group on interface wlp1s0.IPv6 with address fe80::32cd:3799:f3e6:f97b.
May  7 14:50:14 surface avahi-daemon[510]: Interface wlp1s0.IPv6 no longer relevant for mDNS.
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9262] dhcp4 (wlp1s0): canceled DHCP transaction
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9263] dhcp4 (wlp1s0): activation: beginning transaction (timeout in 45 seconds)
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9263] dhcp4 (wlp1s0): state changed no lease
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9266] dhcp6 (wlp1s0): canceled DHCP transaction
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9267] dhcp6 (wlp1s0): activation: beginning transaction (timeout in 45 seconds)
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9267] dhcp6 (wlp1s0): state changed no lease
May  7 14:50:14 surface systemd-resolved[425]: wlp1s0: Bus client reset search domain list.
May  7 14:50:14 surface avahi-daemon[510]: Withdrawing address record for 192.168.0.201 on wlp1s0.
May  7 14:50:14 surface avahi-daemon[510]: Leaving mDNS multicast group on interface wlp1s0.IPv4 with address 192.168.0.201.
May  7 14:50:14 surface avahi-daemon[510]: Interface wlp1s0.IPv4 no longer relevant for mDNS.
May  7 14:50:14 surface systemd-resolved[425]: wlp1s0: Bus client set default route setting: no
May  7 14:50:14 surface systemd-resolved[425]: wlp1s0: Bus client reset DNS server list.
May  7 14:50:14 surface gnome-shell[6272]: cr_parser_new_from_buf: assertion 'a_buf && a_len' failed
May  7 14:50:14 surface gnome-shell[6272]: cr_declaration_parse_list_from_buf: assertion 'parser' failed
May  7 14:50:14 surface NetworkManager[515]: <info>  [1651956614.9483] device (wlp1s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
May  7 14:50:15 surface update-notifier[8585]: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
May  7 14:50:15 surface update-notifier[8585]: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
May  7 14:50:15 surface wpa_supplicant[575]: p2p-dev-wlp1s0: CTRL-EVENT-DSCP-POLICY clear_all
May  7 14:50:15 surface wpa_supplicant[575]: p2p-dev-wlp1s0: CTRL-EVENT-DSCP-POLICY clear_all
May  7 14:50:15 surface wpa_supplicant[575]: nl80211: deinit ifname=p2p-dev-wlp1s0 disabled_11b_rates=0
May  7 14:50:15 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-DSCP-POLICY clear_all
May  7 14:50:15 surface gnome-shell[6272]: An active wireless connection, in infrastructure mode, involves no access point?
May  7 14:50:15 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-DSCP-POLICY clear_all
May  7 14:50:15 surface wpa_supplicant[575]: nl80211: deinit ifname=wlp1s0 disabled_11b_rates=0
May  7 14:50:15 surface gnome-shell[6272]: cr_parser_new_from_buf: assertion 'a_buf && a_len' failed
May  7 14:50:15 surface gnome-shell[6272]: cr_declaration_parse_list_from_buf: assertion 'parser' failed
May  7 14:50:16 surface systemd[1]: Reached target Sleep.
May  7 14:50:16 surface systemd[1]: Starting Record successful boot for GRUB...
May  7 14:50:16 surface systemd[1]: Starting System Suspend...
May  7 14:50:16 surface systemd-sleep[25260]: Entering sleep state 'suspend'...
May  7 14:50:16 surface kernel: [102086.912491] PM: suspend entry (s2idle)
May  7 14:50:16 surface kernel: [102086.924445] Filesystems sync: 0.011 seconds
May  7 14:50:16 surface systemd[1]: grub-common.service: Deactivated successfully.
May  7 14:50:16 surface systemd[1]: Finished Record successful boot for GRUB.
May  7 14:50:16 surface systemd[1]: Starting GRUB failed boot detection...
May  7 14:50:16 surface systemd[1]: grub-initrd-fallback.service: Deactivated successfully.
May  7 14:50:16 surface systemd[1]: Finished GRUB failed boot detection.
May  7 14:50:19 surface kernel: [102087.200755] Freezing user space processes ... (elapsed 0.007 seconds) done.
May  7 14:50:19 surface kernel: [102087.208091] OOM killer disabled.
May  7 14:50:19 surface kernel: [102087.208094] Freezing remaining freezable tasks ... (elapsed 0.001 seconds) done.
May  7 14:50:19 surface kernel: [102087.209936] printk: Suspending console(s) (use no_console_suspend to debug)
May  7 14:50:19 surface kernel: [102087.530459] intel_pch_thermal 0000:00:14.2: CPU-PCH is cool [34C], continue to suspend
May  7 14:50:19 surface kernel: [102087.713501] ACPI: EC: interrupt blocked
May  7 14:50:19 surface kernel: [102089.921084] ACPI: EC: interrupt unblocked
May  7 14:50:19 surface kernel: [102090.353971] OOM killer enabled.
May  7 14:50:19 surface kernel: [102090.353976] Restarting tasks ... 
May  7 14:50:19 surface kernel: [102090.354566] mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_component_ops [i915])
May  7 14:50:19 surface systemd-resolved[425]: Clock change detected. Flushing caches.
May  7 14:50:19 surface kernel: [102090.381014] done.
May  7 14:50:19 surface kernel: [102090.382600] thermal thermal_zone7: failed to read out thermal zone (-61)
May  7 14:50:19 surface bluetoothd[703]: Controller resume with wake event 0x0
May  7 14:50:19 surface systemd-sleep[25260]: System returned from sleep state.
May  7 14:50:19 surface kernel: [102090.401930] PM: suspend exit
May  7 14:50:19 surface goa-daemon[6105]: goa_http_client_check() failed: 2 — Error resolving “onebeam.us”: Temporary failure in name resolution
May  7 14:50:19 surface goa-daemon[6105]: /org/gnome/OnlineAccounts/Accounts/account_1651785886_0: Setting AttentionNeeded to TRUE because EnsureCredentials() failed with: Invalid password with username “admin” (goa-error-quark, 0): Cannot resolve hostname (goa-error-quark, 4)
May  7 14:50:19 surface tracker-miner-f[6070]: g_str_has_suffix: assertion 'str != NULL' failed
May  7 14:50:19 surface tracker-miner-f[6070]: g_str_has_prefix: assertion 'str != NULL' failed
May  7 14:50:19 surface tracker-miner-f[6070]: g_str_has_suffix: assertion 'str != NULL' failed
May  7 14:50:19 surface tracker-miner-f[6070]: g_str_has_prefix: assertion 'str != NULL' failed
May  7 14:50:19 surface systemd[1]: systemd-suspend.service: Deactivated successfully.
May  7 14:50:19 surface systemd[1]: Finished System Suspend.
May  7 14:50:19 surface systemd[1]: Stopped target Sleep.
May  7 14:50:19 surface systemd[1]: Reached target Suspend.
May  7 14:50:19 surface systemd[1]: Stopped target Suspend.
May  7 14:50:19 surface ModemManager[615]: <info>  [sleep-monitor] system is resuming
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.5919] manager: sleep: wake requested (sleeping: yes  enabled: yes)
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.5920] device (wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.8385] device (p2p-dev-wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.8420] manager: NetworkManager state is now DISCONNECTED
May  7 14:50:19 surface kernel: [102090.801592] mmc0: cannot verify signal voltage switch
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.8981] device (wlp1s0): supplicant interface state: internal-starting -> disconnected
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.8982] device (p2p-dev-wlp1s0): state change: unavailable -> unmanaged (reason 'removed', sys-iface-state: 'removed')
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.9001] Wi-Fi P2P device controlled by interface wlp1s0 created
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.9010] manager: (p2p-dev-wlp1s0): new 802.11 Wi-Fi P2P device (/org/freedesktop/NetworkManager/Devices/11)
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.9029] device (p2p-dev-wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.9053] device (wlp1s0): state change: unavailable -> disconnected (reason 'supplicant-available', sys-iface-state: 'managed')
May  7 14:50:19 surface NetworkManager[515]: <info>  [1651956619.9070] device (p2p-dev-wlp1s0): state change: unavailable -> disconnected (reason 'none', sys-iface-state: 'managed')
May  7 14:50:19 surface gnome-shell[6272]: Removing a network device that was not added
May  7 14:50:20 surface goa-daemon[6105]: goa_http_client_check() failed: 2 — Error resolving “onebeam.us”: Temporary failure in name resolution
May  7 14:50:22 surface ModemManager[615]: <info>  [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:1c.0/0000:01:00.0': not supported by any plugin
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-REGDOM-CHANGE init=DRIVER type=COUNTRY alpha2=US
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3236] policy: auto-activating connection 'pebble-5G' (5ecbd87b-7c2f-4bdf-b8b4-ca3f38e0e768)
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3250] device (wlp1s0): Activation: starting connection 'pebble-5G' (5ecbd87b-7c2f-4bdf-b8b4-ca3f38e0e768)
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3255] device (wlp1s0): state change: disconnected -> prepare (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3261] manager: NetworkManager state is now CONNECTING
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3275] device (wlp1s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3283] device (wlp1s0): Activation: (wifi) access point 'pebble-5G' has security, but secrets are required.
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3283] device (wlp1s0): state change: config -> need-auth (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3285] sup-iface[9fc7f1b0d0af9855,8,wlp1s0]: wps: type pbc start...
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3299] device (wlp1s0): state change: need-auth -> prepare (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3302] device (wlp1s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3305] device (wlp1s0): Activation: (wifi) connection 'pebble-5G' has security, and secrets exist.  No new secrets needed.
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3306] Config: added 'ssid' value 'pebble-5G'
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3306] Config: added 'scan_ssid' value '1'
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3306] Config: added 'bgscan' value 'simple:30:-70:86400'
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3306] Config: added 'key_mgmt' value 'WPA-PSK WPA-PSK-SHA256 FT-PSK SAE FT-SAE'
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3306] Config: added 'auth_alg' value 'OPEN'
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3307] Config: added 'psk' value '<hidden>'
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: WPS-CANCEL 
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: SME: Trying to authenticate with dc:bf:e9:ff:26:12 (SSID='pebble-5G' freq=5805 MHz)
May  7 14:50:23 surface kernel: [102094.258155] wlp1s0: authenticate with dc:bf:e9:ff:26:12
May  7 14:50:23 surface kernel: [102094.267749] wlp1s0: send auth to dc:bf:e9:ff:26:12 (try 1/3)
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3550] device (wlp1s0): supplicant interface state: disconnected -> authenticating
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3550] device (p2p-dev-wlp1s0): supplicant management interface state: disconnected -> authenticating
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: Trying to associate with dc:bf:e9:ff:26:12 (SSID='pebble-5G' freq=5805 MHz)
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3840] device (wlp1s0): supplicant interface state: authenticating -> associating
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.3840] device (p2p-dev-wlp1s0): supplicant management interface state: authenticating -> associating
May  7 14:50:23 surface kernel: [102094.296700] wlp1s0: authenticated
May  7 14:50:23 surface kernel: [102094.304148] wlp1s0: associate with dc:bf:e9:ff:26:12 (try 1/3)
May  7 14:50:23 surface kernel: [102094.305480] wlp1s0: RX AssocResp from dc:bf:e9:ff:26:12 (capab=0x1011 status=0 aid=5)
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: Associated with dc:bf:e9:ff:26:12
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-SUBNET-STATUS-UPDATE status=0
May  7 14:50:23 surface kernel: [102094.326660] wlp1s0: associated
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.4197] device (wlp1s0): supplicant interface state: associating -> 4way_handshake
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.4198] device (p2p-dev-wlp1s0): supplicant management interface state: associating -> 4way_handshake
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: WPA: Key negotiation completed with dc:bf:e9:ff:26:12 [PTK=CCMP GTK=CCMP]
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-CONNECTED - Connection to dc:bf:e9:ff:26:12 completed [id=0 id_str=]
May  7 14:50:23 surface kernel: [102094.393609] IPv6: ADDRCONF(NETDEV_CHANGE): wlp1s0: link becomes ready
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.5008] device (wlp1s0): supplicant interface state: 4way_handshake -> completed
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.5008] device (wlp1s0): Activation: (wifi) Stage 2 of 5 (Device Configure) successful. Connected to wireless network "pebble-5G"
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.5009] device (p2p-dev-wlp1s0): supplicant management interface state: 4way_handshake -> completed
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.5012] device (wlp1s0): state change: config -> ip-config (reason 'none', sys-iface-state: 'managed')
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-SIGNAL-CHANGE above=0 signal=-51 noise=9999 txrate=234000
May  7 14:50:23 surface NetworkManager[515]: <info>  [1651956623.5021] dhcp4 (wlp1s0): activation: beginning transaction (timeout in 45 seconds)
May  7 14:50:23 surface avahi-daemon[510]: Joining mDNS multicast group on interface wlp1s0.IPv6 with address fe80::32cd:3799:f3e6:f97b.
May  7 14:50:23 surface avahi-daemon[510]: New relevant interface wlp1s0.IPv6 for mDNS.
May  7 14:50:23 surface avahi-daemon[510]: Registering new address record for fe80::32cd:3799:f3e6:f97b on wlp1s0.*.
May  7 14:50:23 surface dbus-daemon[514]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.206' (uid=1000 pid=6272 comm="/usr/bin/gnome-shell ")
May  7 14:50:23 surface systemd[1]: Starting Fingerprint Authentication Daemon...
May  7 14:50:23 surface dbus-daemon[514]: [system] Successfully activated service 'net.reactivated.Fprint'
May  7 14:50:23 surface systemd[1]: Started Fingerprint Authentication Daemon.
May  7 14:50:23 surface gnome-shell[6272]: JS ERROR: Failed to initialize fprintd service: Gio.IOErrorEnum: GDBus.Error:net.reactivated.Fprint.Error.NoSuchDevice: No devices available#012asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:114:23
May  7 14:50:23 surface wpa_supplicant[575]: wlp1s0: CTRL-EVENT-SIGNAL-CHANGE above=1 signal=-51 noise=9999 txrate=234000
May  7 14:50:24 surface goa-daemon[6105]: goa_http_client_check() failed: 2 — Error resolving “onebeam.us”: Temporary failure in name resolution
</details>

**Comment by lsitongia** [comment]
Same thing happens when I am installing software with the "Ubuntu Software" app: when it prompts for my password, the on screen keyboard doesn't work.

**Comment by lsitongia** [comment]
I installed Ubuntu 21.10 plus all the surface-specific stuff (kernel, etc.) and this is no longer a problem. Seems to be a bug with Ubuntu 22.40.

**Comment by lsitongia** [comment]
Also, the keyboard slides up from the bottom in 21.10, although one not know that because you have to slide up slowly, with finese.

**Comment by lsitongia** [comment]
As addressed above.
